Gopkg.toml

```diff
@@ -16,7 +16,7 @@ required = [
 # TODO switch to tags before PMM 2.0 release
 [[constraint]]
   name = "github.com/percona/pmm"
-  branch = "PMM-2.0"
+  branch = "PMM-3470-change-apis"
```

Comment:
There is no PR for that branch.

Reply:
https://github.com/percona/pmm/pull/310

Reply:
Merged, please revert a branch name there.